.github/workflows/flaky-test-monitor-regression.yml

```diff
@@ -0,0 +1,63 @@
+name: Flaky Test Monitor Test
```

Comment:
Sure will do. But to answer your question, it will be run on every PR or on bors branches (staging and trying). This is necessary because the regular flaky test monitor job will actually push data to bigquery, but we don't want to do this for the test.